scripts/smoke_test_examples.py

```
@@ -0,0 +1,388 @@
+                        # For dataset examples, look for data processing success
+                        if is_dataset_example:
+                            for keyword in DATASET_SUCCESS_KEYWORDS:
+                                if keyword in line or keyword.lower() in line_lower:
+                                    dataset_success_detected = True
+                        
+                        # Check for setup success keywords (exact matches)
+                        for keyword in SETUP_SUCCESS_KEYWORDS:
+                            if keyword in line or keyword.lower() in line_lower:
+                                setup_detected = True
+                
+                except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)